### DISCUSSION TITLE
Fantastic, but I need a Windows version

### DISCUSSION THREAD

**Discussion by josephny** [MartinPham/FindMySync#23] [General]
Any chance there is a solution like this one that does not require a Mac PC?

**Comment by SimonBard** [comment]
You can use sickcodes to start a virtual macOS:
https://github.com/sickcodes/Docker-OSX/issues

There was a video of someone using an iphone to get the location, any cheap iphone will do. But he did not explain in detail and did not answer my email. Probably
